nixos/modules/services/web-apps/davis.nix

```
@@ -381,6 +383,7 @@ in
           APP_CACHE_DIR = "${cfg.dataDir}/var/cache";
           APP_LOG_DIR = "${cfg.dataDir}/var/log";
         };
+        phpPackage = lib.mkDefault pkgs.php83;
```

```suggestion
        phpPackage = lib.mkDefault pkgs.php83;
```
It's possible to reference `php` from `davis` package. I'd expose php in `davis.passthru.php` and reference it. Likely there is some other way of doing this, but that's what first came to mind.

---

`phpPackage` is likely not even necessary, since the davis package can be already overridden. So, the inclusion of which php version to use, should be there. Unless, you have some other idea of why this should be here. I don't have context.

---

Hi @superherointj ! :wave: Thanks for the review.

I first tried this by only updating the davis package, but without this `phpPackage` line above, phpfpm would default to php84, which is no good.

So I do believe the `phpPackage` option to phpfpm is required, however if we can reference it via the davis package that would be more ideal!

---

I've pushed changes for this. Let me know if this resolves it to your satisfaction!

---

> phpfpm

Forgive my PHP ignorance. I'm unfamiliar with `phpfpm`. As an external service. You could configure it using Davis's NixOS module and referencing Davis's PHP package.

---

> I've pushed changes for this.

Thanks for your responsiveness.

> Let me know if this resolves it to your satisfaction!

I'm only providing you my opinion as what I would do differently.
I'm no longer a committer. And I can no longer merge code.
After your PR is approved by reviewers. I suggest you to add an active committer for merging it.